src/main/scala/io/sdkman/changelogs/JavaMigrations.scala

```diff
@@ -468,4 +468,14 @@ class JavaMigrations {
       .insert()
     Seq(Linux64, MacOSX, Windows).foreach(removeVersion("java", "13.ea.02-open", _))
   }
+  @ChangeSet(order = "089", id = "089-add_sapmachine_java_11.0.2 author = "msailer")
+  def migrate089(implicit db: MongoDatabase): Unit = {
+    List(
+      Version("java", "11.0.2-sapmachine", "https://github.com/SAP/SapMachine/releases/download/sapmachine-11.0.2/sapmachine-jdk-11.0.2_linux-x64_bin.tar.gz", Linux64),
```

Comment:
We have a 15 character limit on version, so can we go for something shorter?

Reply:
Cool, so I would like to go for "sapmchn" then :)